src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Models/RazorCodeAction.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using MediatR;
+using Newtonsoft.Json;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions.Models
+{
+    [JsonObject]
+    public class RazorCodeAction : CodeAction, IRequest<RazorCodeAction>, IBaseRequest
+    {
+        public RazorCodeAction()
+        {
+        }
+
+        [JsonProperty(PropertyName = "title", Required = Required.Always)]
+        public new string Title { get; set; }
```

Comment:
Reimplements the properties to ensure we have direct control over property (de)serialization. Without this, ran into extensive platform compatibility issues.

Reply:
Would be good to add a code comment about this.